docs/content/doc/installation/from-source.en-us.md

```diff
@@ -15,18 +15,63 @@ menu:
 
 # Installation from source
 
-This section will not include basic [installation instructions](https://golang.org/doc/install).
+## Dependencies
 
-**Note**: Go version 1.8 or higher is required
+**Note**: [Go Programming Language](http://golang.org): Version >= 1.8
 
-## Download
+We are going to create a new user called `git` and install/setup everything under that user:
+
+```sh
+adduser \
+   --system \
+   --shell /bin/bash \
+   --gecos 'Git Version Control' \
+   --group \
+   --disabled-password \
+   --home /home/git \
+   git
+```
+
+## Installing Go
+
+If your system's Go matches the requirements, please skip this section.
+
+### Download
```

Comment:
I don't think we need a tutorial on installing Go, as https://golang.org/doc/install will always be up to date, and we can just link to that.

Reply:
But to run Gitea created a new user, and then I added the construction using the user git